### PR TITLE
[Merged by Bors] - feat(Complex):  `reProdIm` lemmas

### PR DESCRIPTION
**Pull request by javierlcontreras**
From FLT

Co-authored-by: Yaël Dillies <yael.dillies@gmail.com>

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Data/Complex/Basic.lean [hunk 1]
```diff
@@ -58,6 +58,9 @@ theorem eta : ∀ z : ℂ, Complex.mk z.re z.im = z
 theorem ext : ∀ {z w : ℂ}, z.re = w.re → z.im = w.im → z = w
   | ⟨_, _⟩, ⟨_, _⟩, rfl, rfl => rfl
 
+lemma «forall» {p : ℂ → Prop} : (∀ x, p x) ↔ ∀ a b, p ⟨a, b⟩ := by aesop
+lemma «exists» {p : ℂ → Prop} : (∃ x, p x) ↔ ∃ a b, p ⟨a, b⟩ := by aesop
+
 attribute [local ext] Complex.ext
```
Comment (loefflerd) [inline Mathlib/Data/Complex/Basic.lean:64]: tiny gripe: please add your new lemmas after the `attribute [local ext] Complex.ext` line, to keep the attribute tag together with the lemma it refers to.

**Review by loefflerd — COMMENTED** [review]
This looks very reasonable to me. I guess the old name `Complex.Set.reProdIm` was a mistake, the new name is definitely better. Thanks!

maintainer delegate

File: Mathlib/Data/Complex/Basic.lean (hunk 1 above)
Comment (jcommelin) [reply Mathlib/Data/Complex/Basic.lean:64]: @javierlcontreras please make this change, and merge the PR afterwards:

bors d+

Thanks for the lemmas!

**Comment by YaelDillies** [comment]
> I guess the old name `Complex.Set.reProdIm` was a mistake

I can tell you what happened here: It was originally called `set.re_prod_im`, for dot notation. Then at some point notation was added and dot notation became redundant. Then mathport dropped all `_root_` in declaration names, meaning that our definition was accidentally named `Complex.Set.reProdIm`. Given that dot notation isn't useful here anymore, I think `Complex.reProdIm` has become the best option.

**Pull request closed without merging**